docs/UserGuide.adoc

```diff
@@ -105,8 +105,7 @@ Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 d/Junior Management ic/T0249855I
 pwd/9EwciT plvl/1`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 d/Senior Management
-t/criminal
```

Comment:
Lol here need to change the tag and address, criminal like a little doesn't suit our project ar lol..